tests/ZiraLink.Tests/Services/ProjectService_UnitTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+using Moq;
+
+using ZiraLink.Api.Application;
+using ZiraLink.Api.Application.Services;
+using ZiraLink.Api.Application.Tools;
+using ZiraLink.Api.Models.Project.InputModels;
+using ZiraLink.Domain;
+using ZiraLink.Domain.Enums;
+using ZiraLink.Tests.TestData;
+using ZiraLink.Tests.Tools;
+
+namespace ZiraLink.Tests.Services
+{
+    public class ProjectService_UnitTest
+    {
+        private readonly ProjectService _projectService;
+
+        private readonly Mock<ILogger<ProjectService>>? _mockILoggerProjectService;
+        private readonly Mock<IBus>? _mockIBus;
+        private readonly Mock<IHttpClientFactory>? _mockIHttpClientFactory;
+         
+        public ProjectService_UnitTest()
```

Comment:
As a convention, suggested to rename ProjectService_UnitTest to ProjectServiceTests.

If you rename the project name to ZiraLink.UnitTests (from ZiraLink.Tests), the project name will indicate the type of the tests.